ulauncher/ui/windows/ulauncher_window.py

```diff
@@ -301,6 +297,13 @@ def position_window(self) -> None:
             else:
                 self.move(pos_x, pos_y)
 
+        self.window_frame.set_properties(
+            margin_top=margin,
+            margin_bottom=margin,
+            margin_start=margin,
+            margin_end=margin,
```

Comment:
it's actually possible to set margin=margin as a shorthand, but we'd want different values for x and y axis most likely.